components/basic-utilities-list.vue

```diff
@@ -14,7 +15,7 @@
       </v-list-tile-content>
     </v-list-tile>
 
-    <v-list-tile @click="customTimetableDialogOpen = true">
+    <v-list-tile @click="(customTimetableDialogOpen = true) + trackMatomoEvent('Menu','openCustomTimeTableDialog')">
```

Comment:
Was in @click="..." steht ist JS-Code, der "+"-Operator ist wie in Java für die Konkatenation von Strings gedacht, funktioniert aber in JS auch in diesem Zusammenhang. Besser ist es wenn du hier die Befehle einfach durch ein Semikolon trennst, also "Befehl1; Befehl2".